1.0/docs/devguide/styling.md

```diff
@@ -24,92 +40,148 @@ provided via `<style>` tags placed inside the element's local DOM `<template>`.
             display: block;
             border: 1px solid red;
           }
-          #child-element {
+          /* In shady DOM this will become `.child-element.my-element {` */
+          .child-element {
             background: yellow;
           }
-          /* styling elements distributed to content (via ::content) requires */
-          /* selecting the parent of the <content> element for compatibility with */
-          /* shady DOM . This can be :host or a wrapper element. */
-          .content-wrapper ::content > .special {
-            background: orange;
-          }
+
         </style>
 
-        <div id="child-element">In local DOM!</div>
-        <div class="content-wrapper"><content></content></div>
+        <div class="child-element">In local DOM!</div>
+        <content></content>
 
       </template>
 
       <script>
-
           Polymer({
               is: 'my-element'
           });
-
       </script>
 
     </dom-module>
 
-To place styles outside of the element, or share styles between elements, you can create
-a [style module](#style-modules).
+In the page, you might have:
+
+    <my-element>
+        <!-- In shady DOM the class is "child-element style-scope my-element" -->
+        <div class="child-element">In light DOM!</div>
+    </my-element>
+
+And see that only `In local DOM` has a yellow background.
 
 **Note:**  Prior to Polymer 1.1, the recommendation was to place `<style>` tags
 inside the `<dom-module>` for an element (but _outside_ the `<template>`). This
  is still supported, but is no longer recommended.
 {: .alert .alert-info }
 
+**NOTE: I like introducing what shadow DOM styling SHOULD be like, and then explain what we should do with shady DOM, and why.**
+
+#### Shim limitations
```

Comment:
Like these, but how about renaming them to "Shady DOM limitations" since we have both shady DOM and the custom properties shims to discuss in this chapter.